server/test/gifImg.spec.js

```diff
@@ -38,7 +38,7 @@ describe('GIF UPLOAD IMAGE TO CLOUDINARY ',()=>{
         it('It should post an image to cloudinaary and also return the link', (done)=>{
             chai.request(server)
             .post('/v1/gifs')
-            .set("authorization", "Bearer " + token)
```

Comment:
i was formerly having issues with the header. So when i changed to token, the test passed

Reply:
The issue is that, if i do it  this way .set("authorization", "Bearer " + token), it would give me a 401 response which is as a result of an invalid token provided. 
But when i do it this way, 
.set('token', token)
it gives me a 201 response which is OK